pkg/controller/tidbcluster/pod_control.go

```diff
@@ -458,10 +458,15 @@ func (c *PodController) syncTiKVPodForEviction(ctx context.Context, pod *corev1.
 			klog.Infof("Region leader count is %d for Pod %s/%s", leaderCount, pod.Namespace, pod.Name)
 
 			if leaderCount == 0 {
+				if err := kvClient.FlushLogBackupTasks(ctx); err != nil {
+					klog.ErrorS(err, "failed to flush log backup tasks. continue to delete pod...", "pod", pod.Name, "pod.ns", pod.Namespace)
```

Comment:
How about using the same log format as before?

Reply:
Is it OK to ignore this err?

Reply:
Yes, it seems blocking rolling restart is more unacceptible than make log backup has larger gap.